standalone.Dockerfile

```
@@ -25,14 +25,21 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Then, use a final image without uv
 FROM debian:bookworm-slim
 
+# Setup a non-root user
+RUN addgroup --gid 10001 --system nonroot \
+ && adduser  --uid 10000 --system --ingroup nonroot --home /home/nonroot nonroot
```

Consistency with https://github.com/astral-sh/uv-docker-example/pull/65#discussion_r2376304370

```suggestion
RUN groupadd --system --gid 999 nonroot \
 && useradd --system --gid 999 --uid 999 --create-home nonroot
```